### ISSUE TITLE
SVM-DR and indexing limitation

### ISSUE THREAD

**Issue by jasonfiset** [NetAppDocs/bluexp-backup-recovery#162]
### Page URL

https://docs.netapp.com/us-en/bluexp-backup-recovery/reference-limitations.html

### Page title

Known limitations

### Summary

Enabling Indexing on backups from SVM-DR volumes is currently not support.  An error message that will not allow enabling indexing turns up stating that it cannot be enabled because it is backing up a SVM-DR volume.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-tonacki** [comment]
Hi Jason,  Thanks for pointing this out.  So just to confirm, when you want to restore data from SVM-DR volumes you need to use the Browse & Restore mechanism because the Search & Restore mechanism can't be configured (Indexed) on those clusters?   Is that right?

Thanks, Tom

**Comment by jasonfiset** [comment]
Indexing can’t be turned on because of the following error:

“This operation is not permitted on a Vserver that is configured as the destination for Vserver DR.”

***@***.***

This seems to apply to volumes that are backed up in a cascade style relationship when SVM-DR is in use:

Source Volume(s) ----- -> Replication using SVM-DR ------- -> Destination Volume(s) -------- -> Backup using CBS

I’m not sure if we were backing up the Source Volume(s) if we would have run into this issue.  Something we may want to confirm.

Browse and restore along with volume restore continue to function.

Jason Fiset
Technical Solution Specialist

NetApp
778.997.3470 Mobile Phone
***@***.******@***.***>

[NetApp logo]<https://www.netapp.com/>

[NetApp TV link]<https://www.netapp.tv/>[NetApp on Facebook]<https://www.facebook.com/NetApp>[NetApp on Instagram]<https://www.instagram.com/netapp/>[NetApp on YouTube]<https://www.youtube.com/netapp>[NetApp on LinkedIn]<https://www.linkedin.com/company/netapp/>[NetApp on X]<https://twitter.com/NetApp>[NetApp on Discord]<https://netappdiscord.com/>



From: Tom Onacki ***@***.***>
Date: Wednesday, November 29, 2023 at 6:07 AM
To: NetAppDocs/bluexp-backup-recovery ***@***.***>
Cc: Fiset, Jason ***@***.***>, Author ***@***.***>
Subject: Re: [NetAppDocs/bluexp-backup-recovery] SVM-DR and indexing limitation (Issue #162)
EXTERNAL EMAIL - USE CAUTION when clicking links or attachments



Hi Jason, Thanks for pointing this out. So just to confirm, when you want to restore data from SVM-DR volumes you need to use the Browse & Restore mechanism because the Search & Restore mechanism can't be configured (Indexed) on those clusters? Is that right?

Thanks, Tom

**Comment by netapp-tonacki** [comment]
Very clear.  Thanks Jason.  I'll check with the team and see under what conditions this message is displayed.  

Thanks, Tom

**Comment by netapp-tonacki** [comment]
Hi Jason,   We updated the [Limitations topic ](https://docs.netapp.com/us-en/bluexp-backup-recovery/reference-limitations.html#backup-to-object-limitations) a while ago and I forgot to mention it.  We spoke with the PM (Jacob) and we came up with this.  How does that look?

**SVM-DR volume backup is supported with the following restrictions:

- Backups are supported from the ONTAP secondary only.
- The Snapshot policy applied to the volume must be one of the policies recognized by BlueXP backup and recovery, including daily, weekly, monthly, etc. The default "sm_created" policy (used for Mirror All Snapshots) is not recognized and the DP volume will not be shown in the list of volumes that can be backed up.**
